ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -442,6 +442,45 @@ As of AVD 5.0.0 "null" option for esp integrity and encryption has been replaced
 +      encryption: disabled
 ```
 
+### `router_ospf.redistribute.bgp/connected/static.enabled` is required
```

Comment:
### `router_ospf.redistribute.bgp/connected/static` data model has been updated to require the `enabled`

To avoid ambiguous YAML data input, the data model for `router_ospf.redistribute.bgp/connected/static` has been updated to require the `enabled`.

=== "4.x"

    ```yaml
    router_ospf:
      process_ids:
        - id: 400
          redistribute:
            bgp: {}
            connected: {}
            static: {}
    ```

=== "5.x"

    ```yaml
    router_ospf:
      process_ids:
        - id: 400
          redistribute:
            bgp:
             enabled: True
            connected:
             enabled: True
            static:
             enabled: True
    ```

Reply:
The change has been incorporated into `eos_designs` so action is only required when defining `structured_configuration` directly.

Reply:
Fixed!